lib/bambora/v1/profile_resource.rb

```diff
@@ -58,6 +58,48 @@ def create(card_data)
         client.post(path: sub_path, body: card_data, api_key: api_key)
       end
 
+      ##
+      # Get a Bambora payment profile given a customer code.
+      #
+      # @example
+      #
+      #   client = Bambora::Rest::JSONClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   profiles = Bambora::V1::ProfileResource(client: client)
+      #   customer_code = '02355E2e58Bf488EAB4EaFAD7083dB6A'
```

Comment:
Is this a real customer code?

Reply:
It used to be a real API key, not even a customer code 😰. That key was cycled though about a month ago. This no longer represents any live customer code or API key.